packages/commerce-sdk-react/src/hooks/ShopperBaskets/types.ts

```diff
@@ -1,43 +0,0 @@
-/*
- * Copyright (c) 2022, Salesforce, Inc.
- * All rights reserved.
- * SPDX-License-Identifier: BSD-3-Clause
- * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-import {QueryParams} from '../types'
-export interface ShopperBasketParams extends QueryParams {
```

Comment:
nice! we no longer need to manually define the param types! 👍 Saves ton of effort.